css/css-sizing/contain-intrinsic-size/auto-010.html

```diff
@@ -72,7 +72,8 @@
 
   await nextRendering();
   target.className = "cis-auto content-skip";
-  checkSizes([[50, 150]], "Using last remembered size");
+  assert_equals(target.clientWidth, 50, "Using last remembered size");
+  assert_equals(target.clientHeight, 150, "Using last remembered size");
```

Comment:
In case it fails, may be clearer to include "width" or "height" in the message, like

```js
  assert_equals(target.clientWidth, 50, "Using last remembered width");
  assert_equals(target.clientHeight, 150, "Using last remembered height");
```

Or in various of these tests I used this function:

```js
function checkSize(expectedWidth, expectedHeight, msg) {
  assert_equals(target.clientWidth, expectedWidth, msg + " - clientWidth");
  assert_equals(target.clientHeight, expectedHeight, msg + " - clientHeight");
}
checkSize(50, 150, "Using last remembered size");
```

Reply:
Done, thanks:)